### PR TITLE
Adding dockerhub credentials to prevent API rate limit problems.

### PR DESCRIPTION
**Pull request by gguridi**
This change is [Reviewable](https://reviewable.io/reviews/hybridtheory/iab-tcf/2)

### CODE REVIEW

**Comment by gguridi** [comment]
@ahernp This PR is ready to review.

**Review by ahernp — APPROVED** [review]

**Review by ahernp — APPROVED** [review]

**Comment by ahernp** [comment]
I approved the change but cannot merge it beacuse: "Only those with write access to this repository can merge pull requests."

**Pull request merged**
